### PR TITLE
rename e2e-aws to e2e-aws-sdn

### PR DESCRIPTION
**Pull request by jluhrsen**
- this will use the recently added e2e-aws-sdn workflow
which explicitly configures OpenShiftSDN as the CNI. This
is a noop until OpenShiftSDN is no longer the default
CNI, but eventually the plan is for OVNKubernetes to be
the default at which point this explicit configuration
will be needed.

Signed-off-by: Jamo Luhrsen <jluhrsen@gmail.com>

### CODE REVIEW

**Review by vrutkovs — APPROVED** [review]

File: core-services/release-controller/_releases/release-ocp-4.12.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
       "maxRetries": 3,
       "optional": true,
       "prowJob": {
-        "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-aws"
+        "name": "periodic-ci-openshift-release-master-nightly-4.12-e2e-aws-sdn"
```
Comment (stbenjam) [inline core-services/release-controller/_releases/release-ocp-4.12.json:73]: Shouldn't we rename this to aws-sdn above on line 69?

Comment (jluhrsen) [reply core-services/release-controller/_releases/release-ocp-4.12.json:73]: yes. good catch. fixed.

**Pull request merged**